Bottom: 33998152313975aa81f09e82b73c70049813e20f
Top:    6f874f86309af3e51178cd57c5958a08705996ac
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-29 00:36:17 +0100

Refresh of add-a-demonstration-model-for-

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index a20f33c..c0606da 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2300,6 +2300,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     glEnd ();
   }
 #endif
+
   glDepthMask (GL_TRUE);
 
   ghid_draw_bg_image ();
